### ISSUE TITLE
Allow Multiple Users in 'Submitted By' field 

### ISSUE THREAD

**Issue by scottofletcher** [participedia/frontend#814]
This is tricky and NOT A PRIORITY! We discussed having the ability to list multiple users as 'authors' (entry submitters) on a single entry a while back after hearing back from profs/teachers whose students often write entries in groups. The work-arounds have been suboptimal: 1) have the teacher submit it (as in the case of UMontreal and Helene) and then credit the student in the Notes section, 2) have only one student submit and leave a note at the bottom crediting the other students, or 3) create a combined account either as multiple individuals ([example](https://participedia.net/en/people/jordanaugustinrekayi)) or as an institution ([example ](https://participedia.net/en/users/mass-lbp) or [another example](https://participedia.net/en/people/prodep))

All of these work-arounds cause us to a) miss upping our user count, and b) lose the opportunity to connect on an individual basis (ie. individuals are unlikely to spontaneously upload content if they have to send it to another person to upload for them)

**Comment by jesicarson** [comment]
Could possibly treat users like data and create a field similar to methods where you can select from the database of users to add multiple authors. There would still be a "main" author though - ie the one who enters the case and adds the additional authors. 

How often do you think this will happen? Its possible that instances of group work will increase since the TTM committee is more active and doing webinars and such..

**Comment by scottofletcher** [comment]
@jesicarson agreed, I've actually come across quite a few entries that have multiple authors noted at the bottom of the page so would be able to use this feature right away. the point about TTM is also v. valid. just wondering how this would work though - what if someone enters an author by mistake? would this field be like any other in that other users can add or remove authors as they please? could we set up a system where it sends the person being added an email and asks them to confirm that they were a co-author? could confirmed authors be 'locked down'? maybe this should be an admin-only feature and we can make a 'Help' page entry under 'How do I enter a case with multiple authors?' or 'Can more than one person be the author of an entry?'

**Comment by dethe** [comment]
1. This has the danger of getting into the situation we've seen before where we want the users to link only other users who are also in our system as authors, but they also want to link to authors who are not in our system and now we need to keep track of two things.

2. Can we solve this with something like what github does, by automatically linking usernames when prefixed with `@`? So it would still be in the body text, but with links to their user pages? Something like: 

## Additional authors

* @jesicarson
* Gabriel Garcia Marquez
* @scottofletcher 
* Umberto Eco
* @dethe
